### ISSUE TITLE
An idea！In order to detect small targets, two Yolo layers are added to the existing three Yolo layers of V4. Is the bottom layer allowed?

### ISSUE THREAD

**Issue by tianyuluan** [AlexeyAB/darknet#5776]
I try to do this ,add two yolo layers ,the anchors like this:
`5,5, 6,6, 7,7`
`8,8, 9,9, 10,10`
but when i try to train,the terminal tell me that the  anchors is not define.
is there any way to solve that?

**Comment by AlexeyAB** [comment]
> An idea！In order to detect small targets, two Yolo layers are added to the existing three Yolo layers of V4. Is the bottom layer allowed?

Yes, but it will be very slow, look how to do it in yolov3: https://github.com/AlexeyAB/darknet/blob/master/cfg/yolov3_5l.cfg

But better to do: https://github.com/AlexeyAB/darknet#how-to-improve-object-detection

> for training for small objects (smaller than 16x16 after the image is resized to 416x416) - set layers = 23 instead of https://github.com/AlexeyAB/darknet/blob/6f718c257815a984253346bba8fb7aa756c55090/cfg/yolov4.cfg#L895 set stride=4 instead of https://github.com/AlexeyAB/darknet/blob/6f718c257815a984253346bba8fb7aa756c55090/cfg/yolov4.cfg#L892 and set stride=4 instead of https://github.com/AlexeyAB/darknet/blob/6f718c257815a984253346bba8fb7aa756c55090/cfg/yolov4.cfg#L989

**Comment by tianyuluan** [comment]
@AlexeyAB 
hai!Alexey, i have added another two yolo layers ,and should i change the [filters] in the [convolutional] layer befor every yolo layer by the way? :
when there are three yolo layers ,the default is 255 = (80+5)*3 and my classes num is 7 ,when there are five yolo layers ,it is should  be  (7+5)*3 or (7+5)*5  ??
or would you like to explain the meaning of  ''3''   in 255 = (80+5)*3 ..
thanks a lot!!!!

**Comment by AlexeyAB** [comment]
3 - is a number of masks
